### ISSUE TITLE
Combine methode name and doc string in output

### ISSUE THREAD

**Issue by Codeberg-AsGithubAlternative-buhtz** [CleanCut/green#252]
With `-vvv` the output gives the first line of the doc-string of each `test_*()` method like this

```
Green 3.3.0, Coverage 5.5, Python 3.9.2

buhtzology.tests.test_khq
  TestKHK
.   KHQ Question 1

```

When doing `-vv` the output is this.
```
buhtzology.tests.test_khq
  TestKHK
.   test_Q1

```

What I would prefer is to have an output combining the method name and the doc string like this:
```
Green 3.3.0, Coverage 5.5, Python 3.9.2

buhtzology.tests.test_khq
  TestKHK
.   test_Q1: KHQ Question 1

```

Maybe adding a `-vvvv`?

**Comment by CleanCut** [comment]
I like it! Let's do it.

**Comment by CleanCut** [comment]
✅ Included in version `3.4.0` (just released)
